bedrock/careers/templates/careers/404.html

```diff
@@ -38,7 +38,7 @@ <h1 class="error-title">Sorry, we can’t find that job posting</h1>
       </li>
       <li>
         <img class="list-icon" alt="" src="{{ static('protocol/img/icons/default-browser.svg') }}" width="30" height="30">
-        <a href="https://foundation.mozilla.org/?form=donate-404">Donate</a> to the Mozilla Foundation and reclaim the internet from big tech.
+        <a href="{{ donate_url(location='donate-404') }}">Donate</a> to the Mozilla Foundation and reclaim the internet from big tech.
```

Comment:
The values have to be set up on the other end or it won't open the form widget. We can ask them to add a separate one for this page but it will block this merging until that's done, or we can just use the same value on both 404s.